File: ANDROID DEV/admin/app/src/main/java/com/admin/admin/MainActivity.java
package com.admin.admin;
// 1st part of code
import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

import com.google.android.material.card.MaterialCardView;

public class MainActivity extends AppCompatActivity {
    MaterialCardView uploadNotice,uploadImage,addEbook;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
//1st
        uploadNotice = findViewById(R.id.addNotice);
        uploadNotice.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(MainActivity.this, UploadNotice.class));
            }
        });
//2nd
        uploadImage = findViewById(R.id.addImage);
        uploadImage.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(MainActivity.this,UploadImage.class));
            }
        });
//3rd
        addEbook = findViewById(R.id.addEbook);
        addEbook.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(MainActivity.this,UploadPDF.class));
            }
        });


    }
}

File: ANDROID DEV/VideoCall/app/src/main/java/com/horseman/videocall/MainActivity.kt
package com.horseman.videocall

import android.content.Intent
import android.content.pm.PackageManager
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.core.app.ActivityCompat
import com.google.firebase.ktx.Firebase
import com.google.firebase.ktx.initialize
import kotlinx.android.synthetic.main.activity_main.*
import java.time.Instant
import java.util.jar.Manifest

class MainActivity : AppCompatActivity() {

    val permission =
        arrayOf(android.Manifest.permission.CAMERA, android.Manifest.permission.RECORD_AUDIO)
    val requestcode = 1

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        if (!isPermissionGranted()) {
            askPermission()
        }

        Firebase.initialize(this)

       loginBtn.setOnClickListener {
           val username = usernameEdit.text.toString()
           val intent = Intent(this,CallActivity::class.java)
           intent.putExtra("username",username)
           startActivity(intent)
       }
    }

    private fun askPermission() {
        ActivityCompat.requestPermissions(this,permission,requestcode)
    }

    private fun isPermissionGranted(): Boolean {

      permission.forEach {
          if (ActivityCompat.checkSelfPermission(this, it) != PackageManager.PERMISSION_GRANTED)
              return false
      }

        return true

    }


}